web/crux-ui/src/const.ts

```diff
@@ -6,6 +6,7 @@ export const IMAGE_FILTER_MIN_LENGTH = 1 // characters
 export const IMAGE_WS_REQUEST_DELAY = 500 // millis
```

Comment:
In this context, we are currently employing the same approach. Would it be more advantageous to utilize a single common approach? Alternatively, do you foresee any significant differences arising between these methods in the future?

```
export const IMAGE_WS_REQUEST_DELAY = 500 // millis
export const INSTANCE_WS_REQUEST_DELAY = IMAGE_WS_REQUEST_DELAY // millis
export const DEPLOYMENT_EDIT_WS_REQUEST_DELAY = 500 // millis
export const CONFIG_BUNDLE_EDIT_WS_REQUEST_DELAY = 500 // millis```

Reply:
I prefer having separate consts even though they are the same, but if it looks better I could merge them into something like `ENV_EDITOR_THROTTLE` or `EDITOR_WS_REQUEST_DELAY` or something like that.

Reply:
Okay we can leave it, just don't mix the usage like here: `INSTANCE_WS_REQUEST_DELAY = IMAGE_WS_REQUEST_DELAY`, IMHO.